demos/react/froala3/src/index.js

```diff
@@ -64,22 +63,23 @@ const froalaConfig = {
     initialized() {
       // Get and set the editor and wiris versions in this order.
       Generic.setEditorAndWirisVersion(FroalaEditor.VERSION, pluginVersion);        //eslint-disable-line
-      setTimeout(() => {
-        FroalaEditor.INSTANCES[0].html.set(document.getElementById('transform_content').innerHTML);
-      }, 500);
+
+      // Parse the initial content set on the editor through html to render it
+      const contentRendered = WirisPlugin.Parser.initParse(content);
+      FroalaEditor.INSTANCES[0].html.set(contentRendered);
```

Comment:
I'm not sure if this approach would work if there's more than one editor instance on the page.
Have you validate this circumstance?

Reply:
If there is more than one instance, it'll render just the first one. But since this change is for the code of the integration, it can be changed if you add more instances. Also, lots of users will have their own integration code that can be different from ours, so this is just as an example. Maybe i can add a comment to specify it'll only render the first instance of the page, how does it look like?